test/os-cluster.test.ts

```diff
@@ -152,3 +156,14 @@ test('Test Resources with security enabled single-node cluster', () => {
     ],
   });
 });
+
+describe('JSON input validation', () => {
+  test('valid JSON input should parse successfully', () => {
+    expect(() => JSON.parse(validJson)).not.toThrow();
+  });
+
+  test('invalid JSON input should throw an error', () => {
+    const invalidJson = '{name: "John", age: 30}';
+    expect(() => JSON.parse(invalidJson)).toThrow();
+  });
+});
```

Comment:
Test the `additionalConfig` value here. Not the `validJson` since the validation already happened inside the code. @prudhvigodithi WDYT?

Reply:
Couldn't find a way to access the context variable from stack properties. So adding test at parameter level made more sense.

Reply:
`${props.additionalConfig}"` is accessible I believe? Since we are using the same here: https://github.com/opensearch-project/opensearch-cluster-cdk/pull/21/files#diff-d7c4924379311512de5d5b043d465129d2b5fb1254d7a08737f2e3f9b1c13f64R474